tests/test_types.py

```diff
@@ -143,3 +143,9 @@ def test_guess_odp(self):
         self.assertTrue(kind is not None)
         self.assertEqual(kind.mime, 'application/vnd.oasis.opendocument.presentation')
         self.assertEqual(kind.extension, 'odp')
+
+    def test_guess_tar(self):
+        kind = filetype.guess(FIXTURES + '/sample.tar')
```

Comment:
You added this test, but the test for the `sample_gnu.tar` that you added is missing

Reply:
Thanks, @guigarfr for the review!